src/main/java/org/spongepowered/api/item/inventory/ItemStack.java

```diff
@@ -130,4 +132,22 @@
      * @return The level of the enchantment, or -1 if the enchantment is not applied to this stack
      */
     int getEnchantment(Enchantment enchant);
+    
```

Comment:
AttributeHolder is an object with base value that are modified. An ItemStack has no such base values, only modifiers.